jdk/src/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -95,15 +95,11 @@
      *
      * By default, the native crypto is enabled and uses the native
      * crypto library implementation.
-     *
-     * The native crypto for MessageDigest is disabled until
-     * https://github.com/eclipse/openj9/issues/5611 can be resolved.
-     * 
```

Comment:
Please keep this line (101) that separates the paragraphs of the javadoc (as it was before being disabled).

Reply:
done